Update contact sync documentation (#392)
discord-userdoccers/discord-userdoccers: pages/resources/relationships.mdx

| 1     | SPAM    | Friend requests flagged by Discord as spam |
| 2     | IGNORED | Ignored users                              |

<RouteHeader method="POST" url="/users/@me/relationships/bulk">
  Bulk Add Relationships
</RouteHeader>

Adds multiple relationships from [contact sync](/resources/connected-accounts#sync-contacts). May fire multiple [Relationship Add](/topics/gateway-events#relationship-add) Gateway events.

###### JSON Params

| Field    | Type             | Description                                                                |
| -------- | ---------------- | -------------------------------------------------------------------------- |
| user_ids | array[snowflake] | IDs of users to add                                                        |

> | token    | string           | [Contact sync](/resources/connected-accounts#sync-contacts) bulk add token |

Reviewer: ```suggestion
| token    | string           | The [contact sync](/resources/connected-accounts#sync-contacts) bulk add token |
```

###### Response Body

| Field               | Type             | Description                                       |
| ------------------- | ---------------- | ------------------------------------------------- |

> | failed_requests     | array[snowflake] | IDs of users that failed to create a relationship |
> | successful_requests | array[snowflake] | IDs of users that created a relationship          |

Reviewer: ```suggestion
| failed_requests     | array[snowflake] | IDs of the users who could not be friend requested |
| successful_requests | array[snowflake] | IDs of the users who were friend requested |
```
